services/transaction-statistics/tests/functional/requestAll.test.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { ServiceBroker } = require('moleculer');
+
+const requestAll = require('../../shared/utils/requestAll');
+const request = require('../../shared/utils/request');
+const config = require('../../config');
+
+const broker = new ServiceBroker({
+	transporter: config.transporter,
+	logLevel: 'warn',
+	requestTimeout: 15 * 1000,
+	logger: console,
+});
+
+describe('Test requestAll method', () => {
+	beforeAll(async () => {
+		await broker.start();
+		await request.setAppContext({
+			requestRpc: (method, params) => new Promise((resolve, reject) => {
+				broker
+					.call(method, params)
+					.then(res => resolve(res))
+					.catch(err => {
+						console.error(`Error occurred! ${err.message}`);
+						reject(err);
+					});
+			}),
+		});
+	});
+	afterAll(() => broker.stop());
+
+	it('should return proper response', async () => {
+		const generators = await broker.call('indexer.generators');
+		const result = await requestAll(request.requestIndexer, 'generators', {});
```

Comment:
this seems like an unit test since the function only depends on the inputs. Can we move it under unit test instead?